src/components/TaskCard/AddTaskCard.jsx

```diff
@@ -24,7 +24,7 @@ const AddTaskCard = ({ showCreateCard, onCreateCard }) => {
       return;
     }
 
-    onCreateCard(!showCreateCard);
+    onCreateCard(!isCardCreated);
 
```

Comment:
Is there any reason to add extra line break here ? Otherwise can we remove these extra line breaks ?